### PR TITLE
Allow microsecond precision tests

### PR DESCRIPTION
**Pull request by naveen-ram**
IBM i supports microsecond precision timestamps.

Fixes: #48

### CODE REVIEW

**Comment by naveen-ram** [comment]
I am working on merge conflicts on merging master into testing-setup branch, then I will run tests again.

**Comment by kadler** [comment]
Closing/re-opening to re-run CI

**Comment by kadler** [comment]
The microsecond TIME test is failing: https://github.com/IBM/sqlalchemy-ibmi/pull/49/checks?check_run_id=540261947#step:10:350

I saw [this](https://github.com/mkleehammer/pyodbc/wiki/Tips-and-Tricks-by-Database-Platform#time-columns) earlier while looking through the Wiki and I assume that's the cause. Of course, I'm pretty sure that Db2 does not support sub-second precision for TIME columns (only TIMESTAMP) so we probably need to skip this test.

**Comment by kadler** [comment]
https://www.ibm.com/support/knowledgecenter/ssw_ibm_i_74/db2/rbafzch2time.htm

Yep, no sub-seconds for time objects.

File: test/test_suite.py [hunk 1]
```diff
@@ -19,6 +19,8 @@
     import UnicodeTextTest as _UnicodeTextTest
 from sqlalchemy.testing.suite \
     import UnicodeVarcharTest as _UnicodeVarcharTest
+from sqlalchemy.testing.suite \
+    import TimeMicrosecondsTest as _TimeMicrosecondsTest
```
Comment (kadler) [inline test/test_suite.py:23]: The changes to this file should be backed out since they are no longer needed.

File: test/test_suite.py [hunk 2]
```diff
@@ -211,3 +211,4 @@ def test_literal(self):
 
     def test_literal_non_ascii(self):
         return
+
```
Comment (kadler) [inline test/test_suite.py:214]: You missed this too

**Pull request merged**